test_run.sh

```diff
@@ -112,6 +112,16 @@ xmllint_test() {
   find -L ./bazel-testlogs -iname "*.xml" | xargs -n1 xmllint > /dev/null
 }
 
+multiple_junit_suffixes() {
+  bazel test //test:JunitMultipleSuffixes
+  matches=$(grep -c -e 'Running E2E' -e 'Running IT' ./bazel-testlogs/test/JunitMultipleSuffixes/test.log)
+  if [ $matches -eq 2 ]; then
+    return 0
+  else
+    return 1
+  fi
+}
```

Comment:
JUnitCore creates an xml output. I actually don't think we should test this since I didn't add any code relating to it. Additionally when we'll (soon) move to java_test and remove the launcher this test will be even more redundant.
WDYT?